docs/_config.yml

```diff
@@ -15,7 +15,7 @@ links-java-tutorial: http://dev.solace.com/get-started/java-tutorials/topic-queu
 links-solaceCloud-setup: http://cloud.solace.com/create-messaging-service/
 docs-vmr-setup: http://docs.solace.com/Solace-VMR-Set-Up/Setting-Up-VMRs.htm
 docs-core-concepts: http://docs.solace.com/Features/Core-Concepts.htm
-docs-jms-home: http://docs.solace.com/Solace-JMS-API/JMS-home-open.htm
+docs-jms-home: https://docs.solace.com/Solace-JMS-API/jms-get-started-open.htm
```

Comment:
This doesn't seem like the final fix. It's not aligned with the // approach used in other fixes.

Reply:
I think the http:// vs // thing is a template issue, as I don't see any other fixes using // links in the _config.yml.